notebooks/02.05-Protein-Geometry.ipynb

```diff
@@ -145,7 +145,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<img src=\"./Media/init-path.png\" width=\"700\">"
+    "<img src=\"https://github.com/RosettaCommons/PyRosetta.notebooks/blob/master/notebooks/Media/init-path.png?raw=true\" width=\"700\">"
```

Comment:
Again, why pull from the web?  Is this something to do with the colab environ?  Will ALL images across ALL notebooks need to change? If so, is it possible to have a function to get the path?

Reply:
Yeah I can do that. Do you think it's better to have it locally or pulled from the internet?